WhitespaceAfter/all-examples-in-one/config-all-in-one.xml

```diff
@@ -20,7 +20,6 @@
     <module name="TreeWalker">
         <module name="WhitespaceAfter">
             <property name="id" value="example1"/>
-            <property name="tokens" value="COMMA, SEMI"/>
```

Comment:
https://checkstyle.org/checks/whitespace/whitespaceafter.html#WhitespaceAfter

there are 2 examples in this Check
Why we missing example2 ?

Reply:
> https://checkstyle.org/checks/whitespace/whitespaceafter.html#WhitespaceAfter
> 
> there are 2 examples in this Check Why we missing example2 ?

this is from non-compilable source: https://github.com/checkstyle/checkstyle/tree/master/src/xdocs-examples/resources-noncompilable/com/puppycrawl/tools/checkstyle/checks/whitespace

Reply:
<img width="1135" alt="image" src="https://github.com/checkstyle/test-configs/assets/95259545/ee607425-3c4b-43f0-95af-2885eb43501b">

Reply:
We need to take example1 from https://github.com/checkstyle/checkstyle/tree/master/src/xdocs-examples/resources-noncompilable/com/puppycrawl/tools/checkstyle/checks/whitespace/whitespaceafter

Reply:
> We need to take example1 from https://github.com/checkstyle/checkstyle/tree/master/src/xdocs-examples/resources-noncompilable/com/puppycrawl/tools/checkstyle/checks/whitespace/whitespaceafter

Yes, this PR is taking Example1 from this path.